gouvlu/theme/templates/reuse/card.html

```diff
@@ -1,7 +1,7 @@
 {% cache cache_duration, 'reuse-card', reuse.id|string, g.lang_code %}
-<a class="card reuse-card card--with-cover" href="{{ url_for('reuses.show', reuse=reuse) }}" title="{{ reuse.title }}">
+<a class="card reuse-card card--with-cover" href="{{ url_for('reuses.show', reuse=reuse, _external=True) }}" title="{{ reuse.title }}">
     <div class="card-image" >
-        <img alt="{{ reuse.title }}" src="{{ reuse.image|placeholder('reuse') }}" />
+        <img alt="{{ reuse.title }}" src="{{ reuse.image.full(external=True)|placeholder('reuse', external=True) }}" />
```

Comment:
I get the following error when trying to load homepage:
`UndefinedError: 'flask_fs.mongo.ImageReference object' has no attribute 'full'`